### ISSUE TITLE
Enhancement proposal

### ISSUE THREAD

**Issue by michaelaye** [ioam/holoviews#3416]
I think overloading the minus operator to add elements vertically might be a workable shortcut?

So, instead of doing:

```python
layout = elem1 + elem2
layout.cols(1)
```
I suggest this as the shortcut for that:

```python
elem1 - elem2
```

but maybe there are issues with that approach I'm not seeing.
Thoughts?

**Comment by jbednar** [comment]
I've previously proposed `e1 / e2` (e1 "over" e2) for that purpose, but I think anything like that is waiting on the Bokeh Layout work that has yet to come to a conclusion.

**Comment by michaelaye** [comment]
what about now? ;)

**Comment by jbednar** [comment]
The Bokeh layout work is long done, but I'm sure @philippjfr or @jlstevens will come up with some new reason why we can't have good things like this. :-)

**Comment by fleimgruber** [comment]
Was just wondering how to have vertical plot layout (irrespective of syntax). Is this possible yet?

**Comment by jbednar** [comment]
Yes, `.cols(1)` will give you a vertical layout, but what I'm hoping out of `/` or `-` is a compositional layout, combining both vertical and horizontal layouts, with an expression like `(a+b*c)/d + e*f` yielding:

```
 ____   ___   ________
|    | |   | |        |
|  a | |b*c| |        |
|____| |___| |        |
 __________  |  e*f   |
|          | |        |
|     d    | |        |
|__________| |________|
```

Right now, you have to use Panel's Row and Column objects if you want a layout like that, which is fine but not very convenient and shouldn't be necessary for hv users since hv has its own notion of layouts that don't require recourse to an external library.

**Comment by fleimgruber** [comment]
Thanks for the example, this looks very convenient indeed!
